packages/react-core/src/components/ExpandableSection/ExpandableSection.tsx

```diff
@@ -42,7 +42,8 @@ export class ExpandableSection extends React.Component<ExpandableSectionProps, E
     toggleText: '',
     toggleTextExpanded: '',
     toggleTextCollapsed: '',
-    onToggle: (): any => undefined,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    onToggle: (_isOpen): void => undefined,
```

Comment:
Curious why an underscore is used? I kind of expect the compiler would have a conflict here.

Can we rename this as `isOpen`?

Reply:
Yes, we can do that, it's just what WebStorm suggested me to call it.

Reply:
Underscores usually represent unused variables.